.github/dependabot.yaml

```
@@ -0,0 +1,52 @@
+      - "dependencies"
+      - "ci-cd"
+    commit-message:
+      prefix: "chore"
+    ignore:
+      # ANY sbt major bump is a deliberate, manual migration, never an automated one:
+      # plugins must first be republished for the new sbt binary version, and the build
+      # definition itself needs rewriting. Still current now that the build is on sbt 2 —
+      # this guards 2.x -> 3.x exactly as it guarded 1.x -> 2.x. Minor/patch bumps are
+      # unaffected and still arrive automatically.
+      - dependency-name: "org.scala-sbt:sbt"
+        update-types: [ "version-update:semver-major" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)